.github/workflows/build-push-docker-images.yaml

```diff
@@ -0,0 +1,183 @@
+name: Build-Push-Docker-Image
+
+on:
+  push:
+    branches:
+      - build-push-docker-images
+
+jobs:
+  build-push-base-rvt-image:
+
+    name: Build and push RVT base Docker image
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        include:
+          - username: rvt
+            user_uid: 1000
+            user_gid: 1000
+            ubuntu_version: 20.04
```

Comment:
(optional) I'm not sure it's worth using a matrix for defining the user ids.

I don't think that this matrix will ever have more than one entry in it: if someone wants a different user id, we're probably not going to burn compute resources building images for them. The way out of that problem would be to look into the cross thing (see my other comment).

Reply:
Sure thing, I'll have them written below.